### PR TITLE
Contact page translation

### PR DESCRIPTION
**Pull request by gbilgedal**

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -1,4 +1,28 @@
 {
+  "contact": {
+    "contactUs":{
+      "message":["Follow Us",
+      "on social media"],
+      "form":["Contact Us",
```
Comment (gbilgedal) [inline public/locales/en/translation.json]: I know I could do them like nameField, surnameField etc. but i didnt want to be keys to be that long so divided them to logical arrays

**Review by ttugba — APPROVED** [review]

**Review by basil-kawak — COMMENTED** [review]
looking good

File: public/locales/en/translation.json (hunk 1 above)
Comment (louisrli) [reply public/locales/en/translation.json]: you should be doing it that way, the keys being longer is more important than having an array of 10

imagine you coming back to this code in 3 weeks
or imagine if someone besides you was translating (usually translation work is outsourced). 

please do the same for info

you should almost never be using arrays in general for translations like this

Comment (gbilgedal) [reply public/locales/en/translation.json]: done @louisrli

**Review by louisrli — APPROVED** [review]

**Pull request merged**